internal/cli/daemon/daemon.go

```diff
@@ -58,14 +58,25 @@ func NewCommand(srv rpc.ArduinoCoreServiceServer, settings *rpc.Configuration) *
 				})
 				if err != nil {
 					// Should never happen...
-					panic("Failed to set default value for directories.builtin.libraries: " + err.Error())
+					feedback.Fatal(i18n.Tr("Cannot set the configuration key directories.builtin.libraries: %v", err), feedback.ErrGeneric)
```

Comment:
Why this change? it looks unrelated and adds a string that should never happen to the translations.